src/posts/2021-06-14-Everyday_Accessibility.md

```diff
@@ -0,0 +1,121 @@
+
+<figure role="figure" aria-label="Typing out the text in an image makes the content more accessible.">
+	<img alt="Instagram post where the photo is a quote reading 'Whatever is good for your soul do that - Anurup' and the quote's text is also typed in the post." src="/img/posts/2021-06-14-Everyday_Accessibility/instagram_quote.jpg" />
+	<figcaption>Typing out the text in an image makes the content more accessible.</figcaption>
+</figure>
+
+### Use “alt text” features to describe images
+Pictures are significant in social media, but also inaccessible to visually impaired readers unless they have a text-based description. When uploading, find the “image description” or “alternative/alt text” feature in the software to write a description. For sample instructions, review [Alt Text for Social Media](https://symphonyagency.com/alt-text-for-social-media/). 
+
+When adding descriptions, don’t write every detail, just enough to ensure everyone can understand your post and share in the image’s meaning. For examples, read Scott Vinkle’s [Considerations when writing alt text](https://ux.shopify.com/considerations-when-writing-alt-text-a9c1985a8204).
+
+<figure role="figure" aria-label="Twitter's Add Description feature allows adding alt text for visually impaired readers.">
+	<img alt="Two mobile phone screenshots. The first shows a draft Twitter post reading, 'Coolest clouds I've ever seen' with a photo attached. The second shows the Image Description input screen with text reading 'A field of wildflowers with hills in the background and clouds overhead'." src="/img/posts/2021-06-14-Everyday_Accessibility/twitter_alt.png" />
+	<figcaption>Twitter's "Add description" feature allows adding "alt text" for visually impaired readers.</figcaption>
+</figure>
+
+### Limit your emojis
+Emojis are great for emphasis and playfulness, but too many and in unconventional placements can be distracting and hard to read. Keep your emojis to the minimum needed to make your point, group them together, and move them towards the end of any text.
+
+> **When you write:** I 👏 know 👏 that’s 👏 right 👏!!!
+> **Text-to-speech reads:** I \[_clapping hands_\] know \[_clapping hands_\] that’s \[_clapping hands_\] right \[_clapping hands_\] !!!
+
+## Documents
+
+### Use a program’s built-in styling features
+Assistive technology (AT) doesn’t interpret a page or document like people do with their eyes; it reads the underlying code markup. This means simply making something _look_ a certain way falls apart when using AT. When formatting a document or presentation, use the style buttons/options in the program to provide the needed markup; for instance, use the “bulleted list” button rather than just a dash (-) to format a list.
```

Comment:
```suggestion
### Use a program’s built-in styling features

Assistive technology (AT) doesn’t interpret a page or document like people do with their eyes; it reads the underlying code markup. This means simply making something _look_ a certain way falls apart when using AT. When formatting a document or presentation, use the style buttons/options in the program to provide the needed markup; for instance, use the “bulleted list” button rather than just a dash (-) to format a list.
```

Reply:
Can we turn the sentence with a semicolon into two separate sentences/thoughts?

Reply:
Okay, sure. So we'll say something like, "Assistive technology (AT) doesn't interpret a page or document like people do with their eyes. AT is reading the underlying code markup which means simply making something _look_ a certain way is not sufficient." I'll update this and put it in when I upload my other changes to see if it flows better.